kafka_utils/util/metadata.py

```diff
@@ -42,16 +43,27 @@ def get_topic_partition_metadata(hosts):
     return topic_partitions
 
 
-def get_topic_partition_with_error(cluster_config, error):
+def get_topic_partition_with_error(cluster_config, error, get_unavailable_brokers=False):
```

Comment:
Could we use a better name e.g. unavailable_brokers? Get usually is used for method names